TLSharp.Core/Session.cs

```diff
@@ -14,9 +14,18 @@ public interface ISessionStore
 
     public class FileSessionStore : ISessionStore
     {
+        private readonly string _storePath;
+
+        public FileSessionStore(string storePath = "")
```

Comment:
ok, now I prefer this to have a better type, it should be `DirectoryInfo`, not string. (And it should default to null.)